pmc.c

```diff
@@ -272,6 +333,18 @@ static void pmc_show(struct ptp_message *msg, FILE *fp)
 			dds->priority2,
 			cid2str(&dds->clockIdentity),
 			dds->domainNumber);
+
+		/* If we received the full extended IEC 62439-3 dataset */
+		if (mgt->length == 2 + sizeof(struct defaultDS)) {
```

Comment:
It would be beneficial to clarify what '2' means, by having a `#define HDR_OFFSET 2` statement.

Reply:
Added a `#define HDR_SIZE 2`